api/registrations/serializers.py

```diff
@@ -12,13 +12,12 @@ class RegistrationSerializer(NodeSerializer):
     retracted = ser.BooleanField(source='is_retracted', read_only=True,
                                  help_text='Whether this registration has been retracted.')
     date_registered = ser.DateTimeField(source='registered_date', read_only=True, help_text='Date time of registration.')
+    embargo_end_date = HideIfRetraction(ser.SerializerMethodField(help_text='When will embargo be lifted?'))
```

Comment:
To be clear, we're talking specifically about `Node#is_pending_embargo` and `Node#is_pending_registration`? I may missing it, but don't see `pending_registration_approval` anywhere. 

It would be pretty concerning to me if these values are consistently the same (although it could just be a bug in the factory or test case). Do you think you could write a test to document this behavior?